spec/models/notification_permission_spec.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
```

Comment:
This class is currently just two associations ... I'd probably either add some validations to it and add specs for those here, or delete this file if no validations needed.